File: src/main/java/com/examportal/dto/StudentQueryResponseDTO.java
package com.examportal.dto;

import java.time.LocalDateTime;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class StudentQueryResponseDTO extends BaseDTO {
    private String username;
    private String title;
    private String subject;
    private String query;
    private String response;
    private String status;
    private LocalDateTime respondedAt;
}

File: Dotnet3/ExamPortal/DTOs/StudentQueryResponseDTO.cs
namespace ExamPortal.DTOs
{
    public class StudentQueryResponseDTO
    {
        public long Id { get; set; }
        public long StudentId { get; set; }
        public string Username { get; set; } = string.Empty;
        public string Title { get; set; } = string.Empty;
        public string Subject { get; set; } = string.Empty;
        public string Query { get; set; } = string.Empty;
        public string? Response { get; set; }
        public string Status { get; set; } = string.Empty;
        public DateTime CreatedAt { get; set; }
        public DateTime? RespondedAt { get; set; }
    }
}
